### PR TITLE
Possibility to disable autoscroll when reloading data

### PR DESCRIPTION
**Pull request by azpery**

### CODE REVIEW

File: KVKCalendar/Classes/Style.swift [hunk 1]
```diff
@@ -85,6 +85,7 @@ public struct TimelineStyle {
     public var timeColor: UIColor = .systemGray
     public var timeFont: UIFont = .systemFont(ofSize: 12)
     public var scrollToCurrentHour: Bool = true
```
Comment (kvyatkovskys) [inline KVKCalendar/Classes/Style.swift]: This is the same property to disable/enable auto scroll 🙂

Comment (azpery) [reply KVKCalendar/Classes/Style.swift]: I've tryed it and actually, it scrolls to top if this property is set to false. (I comment the line that is responsible for this behaviour :p )

File: KVKCalendar/Classes/TimelineView.swift [hunk 2]
```diff
@@ -396,6 +396,10 @@ final class TimelineView: UIView, CompareEventDateProtocol {
     }
     
     private func scrollToCurrentTime(startHour: Int) {
+        guard style.timeline.enableAutoScroll else {
+            return
+        }
+        
         guard isEnabledAutoScroll else {
             isEnabledAutoScroll = true
             return
```
Comment (azpery) [inline KVKCalendar/Classes/TimelineView.swift]: @kvyatkovskys Line 405 - 409, "scrollView.setContentOffset(.zero, animated: true)", it scrolls to top

Comment (kvyatkovskys) [reply KVKCalendar/Classes/TimelineView.swift]: yep, we can try this 
Screenshot 2020-07-22 at 12 10 19
instead line 399 - 401

to use one property

Comment (azpery) [reply KVKCalendar/Classes/TimelineView.swift]: Ok yes, indeed, it would be better. I commit the new changes :)

**Comment by kvyatkovskys** [comment]
@azpery can you resolve conflicts, pls? You need to rebase your branch with `feature/v_0.3.4`

**Comment by azpery** [comment]
@kvyatkovskys I rebased my branch with upstream. There are no conflicts but your commits are in my branch now.

**Comment by kvyatkovskys** [comment]
> @kvyatkovskys I rebased my branch with upstream. There are no conflicts but your commits are in my branch now.

Cool 👍🏼, don't worry about it.

**Review by kvyatkovskys — APPROVED** [review]

**Pull request merged**